cmd/version/version.go

```diff
@@ -117,8 +117,12 @@ func checkForLatestVersion(currVersion string) {
 		return
 	}
 
+	currversion := data["spec"].(map[string]interface{})["version"].(string)
```

Comment:
The [data](https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/plugins/openebs.yaml) contains latest details about the openebsctl deployment distributed via krew.
We are already getting `currVersion` from the function argument. 
```suggestion

```

Reply:
ok then I am removing this.